### PR TITLE
Gh pages upgrade twilio libs

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Gh pages updates.
- upgrade twilio libs.

### CODE REVIEW

File: tutorial/index.jade [hunk 1]
```diff
@@ -38,7 +38,7 @@
       ---
 
       ** See also **
-      * [Twilio API Access Tokens](//www.twilio.com/docs/api/rest/access-tokens)
+      * [Twilio API Access Tokens](//twilio.com/docs/api/rest/access-tokens)
 
   .step(data-title='Token Generation Controller',
         data-file='src/main/java/com/twilio/chat/TokenServlet.java',
```
Comment (acamino) [inline tutorial/index.jade:44]: I'd update the highlight to _26-44_.

**Review by acamino — COMMENTED** [review]
Thanks @juancarlospaco! This one is almost ready - Seems like you missed the changes for [tutorial/config.json](https://github.com/TwilioDevEd/twiliochat-servlets/blob/gh-pages/tutorial/config.json#L4).

**Comment by ghost** [comment]
Oh that file get away, I already fixed it, thanks @acamino :grey_exclamation: 
*Let me know if anything else...*

**Comment by acamino** [comment]
Thanks @juancarlospaco. LGTM

**Pull request merged**